test_schemas/cy/test_language_hub.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
I fixed and updated all of test_languages for all languages only to find validator wouldn't allow me to have a final summary and a hub, so I had to add a separate schema, doh. It seems daft not to include the changes to test_languages even thought they weren't in the end needed

Reply:
Having considered this a bit more, I don't think we should have two test language schemas. I think it's enough to have one schema that uses a hub, rather than a final summary. The custom submission text translations will be verified by the hub schema. When the final summary is updated to use the custom submission text, we can test the English version, and should have confidence that the submission elements will be translated (as they are tested as part of the hub schema).